mtaos_sim/Dockerfile

```diff
@@ -14,27 +14,9 @@ WORKDIR ${LSST_USER_HOME}
 
 # Download the required repos and checkout correct versions
 RUN cd ${AOS_REPOS} \
-    && cd ts_xml/ \
-    && git checkout 74ded62 sal_interfaces/MTM1M3/MTM1M3_Commands.xml \
-    && git checkout f57e9d1 sal_interfaces/MTM1M3/MTM1M3_Events.xml \
-    && cd .. \
-    && git clone --branch develop https://github.com/lsst-ts/ts_config_mttcs \
-    && cd ts_config_mttcs/ \
-    && git checkout 2924779 \
-    && cd .. \
-    && git clone --branch develop https://github.com/lsst-ts/ts_MTAOS.git \
+    && git clone --branch master https://github.com/lsst-ts/ts_MTAOS.git \
     && cd ts_MTAOS/ \
-    && git checkout f9de1d7
-
-# Install the required packages
-COPY setupDocker.sh ${LSST_USER_HOME}
-RUN source ${LSST_USER_HOME}/setupDocker.sh \
-    && make_idl_files.py MTAOS Hexapod MTM1M3 MTM2 \
-    && cd ${AOS_REPOS}/ts_config_mttcs/ \
-    && setup -k -r .\
-    && cd ${AOS_REPOS}/ts_MTAOS/ \
-    && setup -k -r . \
-    && scons
+    && git checkout 2e23b60
```

Comment:
if you make the branch an argument you would probably not need this bit.

Reply:
This is intentional to expose the repo version (branch + commit) to be consistent with the README.md file for the maintenance.